core/lib/rom/relation.rb

```diff
@@ -81,6 +84,30 @@ class Relation
       schema.any?(&:read?) ? schema.to_output_hash : NOOP_OUTPUT_SCHEMA
     }
 
+    # @!attribute [r] mappers
+    #   @return [MapperRegistry] an optional mapper registry (empty by default)
+    option :mappers, default: -> { MapperRegistry.new }
+
+    # @!attribute [r] auto_struct
+    #   @return [TrueClass,FalseClass] Whether or not tuples should be auto-mapped to structs
+    #   @api private
+    option :auto_struct, reader: true, default: -> { false }
```

Comment:
Ahh good point. TBH I didn't pay much attention to method's visibility but I'll address this eventually.